fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowViewCommand.java

```diff
@@ -0,0 +1,148 @@
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.trees.plans.commands;
+
+import org.apache.doris.catalog.Column;
+import org.apache.doris.catalog.Database;
+import org.apache.doris.catalog.Env;
+import org.apache.doris.catalog.ScalarType;
+import org.apache.doris.catalog.Table;
+import org.apache.doris.catalog.View;
+import org.apache.doris.common.ErrorCode;
+import org.apache.doris.common.ErrorReport;
+import org.apache.doris.common.util.Util;
+import org.apache.doris.mysql.privilege.PrivPredicate;
+import org.apache.doris.nereids.NereidsPlanner;
+import org.apache.doris.nereids.StatementContext;
+import org.apache.doris.nereids.parser.NereidsParser;
+import org.apache.doris.nereids.properties.PhysicalProperties;
+import org.apache.doris.nereids.trees.plans.PlanType;
+import org.apache.doris.nereids.trees.plans.commands.info.TableNameInfo;
+import org.apache.doris.nereids.trees.plans.logical.LogicalPlan;
+import org.apache.doris.nereids.trees.plans.visitor.PlanVisitor;
+import org.apache.doris.nereids.util.PlanUtils;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.OriginStatement;
+import org.apache.doris.qe.ShowResultSet;
+import org.apache.doris.qe.ShowResultSetMetaData;
+import org.apache.doris.qe.StmtExecutor;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * ShowViewCommand
+ */
+public class ShowViewCommand extends ShowCommand {
+    private static final ShowResultSetMetaData META_DATA = ShowResultSetMetaData.builder()
+            .addColumn(new Column("View", ScalarType.createVarchar(30)))
+            .addColumn(new Column("Create View", ScalarType.createVarchar(65535)))
+            .build();
+    private String db;
+    private TableNameInfo tbl;
+    private List<View> matchViews = Lists.newArrayList();
+
+    public ShowViewCommand(String db, TableNameInfo tbl) {
+        super(PlanType.SHOW_VIEW_COMMAND);
+        this.db = db;
+        this.tbl = tbl;
+    }
+
+    @Override
+    public <R, C> R accept(PlanVisitor<R, C> visitor, C context) {
+        return visitor.visitShowViewCommand(this, context);
+    }
+
+    private void validate(ConnectContext ctx) throws Exception {
+        if (tbl == null) {
+            ErrorReport.reportAnalysisException(ErrorCode.ERR_NO_TABLES_USED);
+        }
+        if (!Strings.isNullOrEmpty(db)) {
+            // if user specify the `from db`, overwrite the db in `tbl` with this db.
+            // for example:
+            //      show view from db1.tbl1 from db2;
+            // will be rewrote to:
+            //      show view from db2.tbl1;
+            // this act same as in MySQL
+            tbl.setDb(db);
```

Comment:
in syntax, we use multi identifier as table name. if we should set db on tbl, should we only allow identifier as table name? what does `from db` really mean? it indicate table from this db or view from this db? how to show db_a.views that use db_b.tbl?

Reply:
keep the behavior same as old planner